src/Tribe/Data_API.php

```diff
@@ -186,19 +185,21 @@ public function get_event_ids( $post_id ) {
 	/**
 	 * Return Ticket Provider by Order, Product, Attendee, or Ticket ID.
 	 *
-	 * @param $post_id
+	 * @since 4.5
+	 * @since TBD Use new helper method to account for possibly inactive ticket provider.
 	 *
-	 * @return bool|object
+	 * @param int|string $post_id
+	 *
+	 * @return Tribe__Tickets__Tickets|false Ticket provider instance or False if provider is not active.
 	 */
 	public function get_ticket_provider( $post_id ) {
 		$services = $this->detect_by_id( $post_id );
 
-		// if no module class return
-		if ( empty( $services['class'] ) || ! class_exists( $services['class'] ) ) {
+		if( empty( $services['class'] ) ) {
```

Comment:
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after IF keyword; 0 found
* :no_entry_sign: There must be a space between if and the first parenthesis '('

<sub>hash: 8461048d72e3a7e22fdf5e47ab545761</sub>